.github/workflows/testing.yml

```diff
@@ -0,0 +1,117 @@
+name: CloudSDK deployment and testing
+
+env:
+  PR_NUMBER: ${{ github.event.number }}
+  # AWS credentials
+  AWS_EKS_NAME: tip-wlan-main
+  AWS_DEFAULT_OUTPUT: json
+  AWS_DEFAULT_REGION: us-east-2
+  AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  # Cloud SDK certs
+  # CACERT: ${{ secrets.CACERT }}
+  # CAKEY: ${{ secrets.CAKEY }}
+
+on:
+  pull_request:
+     branches: [ master ]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout required repos
+      uses: actions/checkout@v2
+      with:
+        path: wlan-pki-cert-scripts
+        repository: Telecominfraproject/wlan-pki-cert-scripts
+    - name: Checkout Cloud SDK repo
+      uses: actions/checkout@v2
+      with:
+        path: wlan-cloud-helm
+        repository: Telecominfraproject/wlan-cloud-helm
+    - name: Checkout helm values repo
+      uses: actions/checkout@v2
+      with:
+        path: Toolsmith
+        repository: Telecominfraproject/Toolsmith
+        token: ${{ secrets.PAT_TOKEN }}
+        ref: second-cloudsdk-instance-config
+
+    - name: Generate Helm values file
+      run: |
+        #mkdir ./wlan-helm/tip-wlan/resources/certs
+        #mkdir -p ./wlan-pki/testCA/private
+        #mkdir -p ./wlan-pki/testCA/newcerts
+        #mkdir -p ./wlan-pki/generated
+        #touch ./wlan-pki/testCA/index.txt
+        #echo "01" > ./wlan-pki/testCA/serial.txt
+        #echo "${{ env.CACERT }}" | base64 -d > ./wlan-pki/testCA/cacert.pem
+        #echo "${{ env.CAKEY }}" | base64 -d > ./wlan-pki/testCA/private/cakey.pem
+        ./Toolsmith/helm-values/aws-cicd-pr-deployment.yaml.sh ${{ env.PR_NUMBER }} > pr-deployment.yaml
```

Comment:
@brennerm you don't need to have a template on-top of a helmfile... instead, you can use Helmfile's builtin functions, for example, `{{ requiredEnv "PR_NUMBER" }}` in this specific case.